packages/1155-contracts/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @zoralabs/zora-1155-contracts
 
+## 2.1.1-premint-api.0
```

Comment:
since we reverted to the previous version for this package, we can remove this bit from the changelog